index.html

```diff
@@ -287,24 +299,27 @@ <h3>
             Other non-normative documents may be created such as:
           </p>
           <ul>
-            <li>A use case and requirements document</li>
+            <li>A specification for the variable description vocabulary (VVD)</li>
```

Comment:
This one is intended as a Recommendation, is it not? So it should be listed in the "normative deliverables" section

Reply:
Agreed, will move to normative deliverables in the next PR